.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -51,6 +56,13 @@ jobs:
        git config --global user.email "github-actions[bot]@users.noreply.github.com"
        git config --global user.name "GitHub Actions"
 
+    - name: Set GH_TOKEN for standalone execution
+      id: set-gh-token
+      run: |
+        if [ -z "${{ env.GH_TOKEN }}" ]; then
+          echo "GH_TOKEN=${{ secrets.GHA_SECRET_PAT }}" >> $GITHUB_ENV
```

Comment:
Can we have somewhere explained/commented what this value (`GHA_SECRET_PAT`) is and when is this set? If I understand this correctly, this is something that `ide-developer` account has configured in it's secrets? Also, if I run this via my account, what will happen? You explained Jiri above that we don't need to care, though, I don't follow how it's being propagated from my account? Excuse my ignorance please :)

Reply:
Right, it needs a comment there. 
The `GHA_SECRET_PAT` comes from the `ide-developer` account. If a user run this and wants the GHA tests  enabled he should config its **personal** `GHA_SECRET_PAT` on his fork.